code/game/machinery/engi_points_vendor.dm

```diff
@@ -452,6 +452,7 @@
 	var/stop = 0
 	var/beat = 7
 	var/duration = 600
+	var/list/available = list()
 	var/static/list/choices = list("Engineering's Basic Beat", "Engineering's Domination Dance", "Engineering's Superiority Shimmy", "Engineering's Ultimate High-Energy Hustle")

```

Comment:
Sort of, I went on IRC to try and figure out why my tiered lists werent working, got talked into using datums instead, someone suggested using datum/song on PiratePad.... three hours of failed tests later I find out that datum/song already exists and made the whole thing borked and... I forgot where I was going with this, it uses datums now, plz merge